src/excmds.ts

```diff
@@ -198,7 +199,7 @@ export async function editor() {
     let tab = await activeTab()
     let selector = await Messaging.messageTab(tab.id, "excmd_content", "getInputSelector", [])
     let url = new URL(tab.url)
-    if (!await Native.nativegate()) return
```

Comment:
We should probably add a pre commit test for this. `!await` is almost always a bug...